README.md

```diff
@@ -190,9 +192,9 @@ class TestAdIdManager extends IAdIdManager {
     appId: Platform.isAndroid
         ? 'ca-app-pub-3940256099942544~3347511713'
         : 'ca-app-pub-3940256099942544~1458002511',
-    bannerId: BannerAd.testAdUnitId,
-    interstitialId: InterstitialAd.testAdUnitId,
-    rewardedId: RewardedAd.testAdUnitId,
+    bannerId: 'ca-app-pub-3940256099942544/6300978111',
+    interstitialId: 'ca-app-pub-3940256099942544/1033173712',
+    rewardedId: 'ca-app-pub-3940256099942544/5224354917',
```

Comment:
Why did u remove test ad ids? Are they no longer supported?

Reply:
Yeah. Latest version removes static testAdUnitId parameters. See the [Admob](https://developers.google.com/admob/flutter/test-ads) and [AdManager](https://developers.google.com/ad-manager/mobile-ads-sdk/flutter/test-ads) documentation for up to date test ad units.